go.mod

```diff
@@ -13,6 +13,7 @@ require (
 	github.com/openshift/api v3.9.1-0.20190401220125-3a6077f1f910+incompatible
```

Comment:
Can you also update this dependency? When I run `go mod vendor` I get this error:
```
go: github.com/openshift/api@v3.9.1-0.20190401220125-3a6077f1f910+incompatible: invalid pseudo-version: preceding tag (v3.9.0) not found
```

You can use the same version that is in ssp-operator:
```
github.com/openshift/api v0.0.0-20200917102736-0a191b5b9bb0 // release-4.5
```

Reply:
I will change this in a new PR.